presto-product-tests/README.md

```diff
@@ -150,6 +186,60 @@ and also the entire test suite:
 Note: SQL Server product-tests use `microsoft/mssql-server-linux` docker container.
 By running SQL Server product tests you accept the license [ACCEPT_EULA](https://go.microsoft.com/fwlink/?LinkId=746388)
 
+### Running test suites
+
+Tests are further organized into suites which contain execution of multiple test groups in multiple different environments:
+
+```
```

Comment:
Please make it a bit more structural.
I suggest you go over a list of commands, ordering from most usable ones.

IMO sth like:

```
Tests are further organized into suites which contain execution of multiple test groups in multiple different environments.

You can list the available suites using `.... suites list` command.

You can describe the suite using ....

You can run all the tests in a suite using .... command.

```

Please do not quote large command outputs.